doc/backends.md

```diff
@@ -0,0 +1,49 @@
+# Backends in openQA
```

Comment:
It's nice that you add a new documentation file but I wonder how it is integrated into the documentation how it is visualized on http://open.qa

Reply:
Ok, so I'll move it to openQA.

Reply:
But then I likely need to rewrite it as asciidoc. So can I just leave it for now here? Maybe we can figure out a solution later which also covers the other documentation files in os-autoinst like architecture.md and networking.md.

Reply:
Yes, I would also prefer if you keep the documentation in os-autoinst. I would be happy if you can tackle the documentation aggregation.

As a side-note, converting from md to asciidoc should not be a problem, you already know [pandoc](https://pandoc.org/), right? ;)

Reply:
Ok, let's see whether generating the documentation can be improved to include files from os-autoinst, too.

Reply:
@Martchus you can simply rename it to asciidoc, I think the rendering would be ok, as there's no weird stuff